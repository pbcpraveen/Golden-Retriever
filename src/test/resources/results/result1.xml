<?xml version="1.0" encoding="UTF-8" standalone="no"?><result><PERSONA>PUBLIUS</PERSONA><PERSONA>CALPURNIA, wife to Caesar.</PERSONA><PERSONA>VOLUMNIUS</PERSONA><PERSONA>CINNA</PERSONA><PERSONA>POPILIUS LENA</PERSONA><PERSONA>LUCIUS</PERSONA><PERSONA>CASCA</PERSONA><PERSONA>DECIUS BRUTUS</PERSONA><PERSONA>MARULLUS</PERSONA><PERSONA>LUCILIUS</PERSONA><PERSONA>OCTAVIUS CAESAR</PERSONA><PERSONA>MESSALA</PERSONA><PERSONA>CICERO</PERSONA><PERSONA>M. AEMILIUS LEPIDUS</PERSONA><PERSONA>Senators, Citizens, Guards, Attendants, &amp;c.</PERSONA><PERSONA>FLAVIUS</PERSONA><PERSONA>VARRO</PERSONA><PERSONA>Young CATO</PERSONA><PERSONA>CLITUS</PERSONA><PERSONA>Another Poet</PERSONA><PERSONA>CINNA, a poet. </PERSONA><PERSONA>LIGARIUS</PERSONA><PERSONA>A Soothsayer</PERSONA><PERSONA>PINDARUS, servant to Cassius.</PERSONA><PERSONA>MARCUS BRUTUS</PERSONA><PERSONA>METELLUS CIMBER</PERSONA><PERSONA>ARTEMIDORUS Of Cnidos, a teacher of rhetoric. </PERSONA><PERSONA>TITINIUS</PERSONA><PERSONA>PORTIA, wife to Brutus.</PERSONA><PERSONA>STRATO</PERSONA><PERSONA>DARDANIUS</PERSONA><PERSONA>MARCUS ANTONIUS</PERSONA><PERSONA>JULIUS CAESAR</PERSONA><PERSONA>TREBONIUS</PERSONA><PERSONA>CASSIUS</PERSONA><PERSONA>CLAUDIUS</PERSONA></result>