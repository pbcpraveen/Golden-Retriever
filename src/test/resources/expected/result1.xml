<?xml version="1.0" encoding="UTF-8" standalone="no"?><result><PERSONA>LIGARIUS</PERSONA><PERSONA>Another Poet</PERSONA><PERSONA>TREBONIUS</PERSONA><PERSONA>CICERO</PERSONA><PERSONA>CASCA</PERSONA><PERSONA>Young CATO</PERSONA><PERSONA>JULIUS CAESAR</PERSONA><PERSONA>CINNA, a poet. </PERSONA><PERSONA>CALPURNIA, wife to Caesar.</PERSONA><PERSONA>DARDANIUS</PERSONA><PERSONA>PINDARUS, servant to Cassius.</PERSONA><PERSONA>CLAUDIUS</PERSONA><PERSONA>M. AEMILIUS LEPIDUS</PERSONA><PERSONA>CLITUS</PERSONA><PERSONA>DECIUS BRUTUS</PERSONA><PERSONA>MARCUS ANTONIUS</PERSONA><PERSONA>FLAVIUS</PERSONA><PERSONA>A Soothsayer</PERSONA><PERSONA>VOLUMNIUS</PERSONA><PERSONA>LUCIUS</PERSONA><PERSONA>MARCUS BRUTUS</PERSONA><PERSONA>POPILIUS LENA</PERSONA><PERSONA>CINNA</PERSONA><PERSONA>ARTEMIDORUS Of Cnidos, a teacher of rhetoric. </PERSONA><PERSONA>OCTAVIUS CAESAR</PERSONA><PERSONA>STRATO</PERSONA><PERSONA>Senators, Citizens, Guards, Attendants, &amp;c.</PERSONA><PERSONA>METELLUS CIMBER</PERSONA><PERSONA>PUBLIUS</PERSONA><PERSONA>TITINIUS</PERSONA><PERSONA>CASSIUS</PERSONA><PERSONA>VARRO</PERSONA><PERSONA>PORTIA, wife to Brutus.</PERSONA><PERSONA>LUCILIUS</PERSONA><PERSONA>MARULLUS</PERSONA><PERSONA>MESSALA</PERSONA></result>